src/pages/editor/EditorInstitution.tsx

```diff
@@ -33,6 +38,23 @@ export const EditorInstitution = () => {
     meta: { errorMessage: t('feedback.error.get_institution') },
   });
 
+  const institutionUsersQuery = useQuery({
+    queryKey: ['institutionUsers', customerId],
+    enabled: !!customerId,
+    queryFn: () => (customerId ? fetchUsers(customerId, [RoleName.Editor, RoleName.InstitutionAdmin]) : undefined),
+    meta: { errorMessage: t('feedback.error.get_users_for_institution') },
+  });
+
+  const institutionUsers = institutionUsersQuery.data ?? [];
+
+  const institutionAdmins = institutionUsers.filter((user) =>
+    user.roles.some((role) => role.rolename === RoleName.InstitutionAdmin)
+  );
+
+  const institutionEditors = institutionUsers.filter((user) =>
+    user.roles.some((role) => role.rolename === RoleName.Editor)
+  );
```

Comment:
Mulig irrelevant spørsmål: Kan en bruker være både admin og redaktør? Skal de da dukke opp to ganger (èn gang i hver liste)?

Reply:
Godt spørsmål! Snakket med PO, og de skal vises på begge plasser.